source/fundamentals/connection.txt

```diff
@@ -27,11 +28,9 @@ sections:
 - :ref:`Connect to MongoDB <connect-to-mongodb>`
 - :ref:`View a List of Connection Options <connection-options>`
 - :ref:`Specify Connection Behavior with the MongoClient Class <specify-mongoclient-settings>`
+- :ref:`Enable Network Compression <network-compression>`
 - :ref:`Enable TLS/SSL on a Connection <tls-ssl>`
 
-.. TODO:(DOCSP-29268) Add enable network compression back
```

Comment:
looks like there's another TODO for this on the `mongoclientsettings.txt` page

Reply:
will update to link correctly